intelmq/bots/outputs/rpz_file/output.py

```diff
@@ -0,0 +1,154 @@
+# -*- coding: utf-8 -*-
+import datetime
+from datetime import datetime
+import os
+from collections import defaultdict
+from pathlib import Path
+
+from intelmq.lib.bot import OutputBot
+
+RPZ_INDICATOR_MAP = {
+    "source.fqdn": "Intel::DOMAIN"
+}
+
+now = datetime.now()
```

Comment:
`now` will be the time when the bot was started. Is that intentional? if yes, please add an inline-comment why.

I'd have expected this in the init or process method.

Reply:
It's for output file timestamp (generate_time, serial).